src/screens/TradeScreen/RightBlock/CreateOrder/CreateOrder.tsx

```diff
@@ -30,8 +30,8 @@ import { useStores } from "@stores";
 import { OrderTypeSheet, OrderTypeTooltip, OrderTypeTooltipIcon } from "./OrderTypeTooltip";
 
 const ORDER_OPTIONS = [
-  { title: "Market", key: ORDER_TYPE.Market, disabled: true },
   { title: "Limit", key: ORDER_TYPE.Limit },
```

Comment:
I am just changing the order of the market and limit order in the ORDER_OPTIONS array. All this does is cause the limit order to load first, instead of the market order, since the market orders are currently disabled.